### ISSUE TITLE
Textfield placeholder not announced in a11y on iOS

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#15188]
On iOS go to the text field demo of the Flutter Gallery. a11y focus on an (empty) text field with a placeholder text. The placeholder text is not read out. Should be read out as "accessibilityLabel", whereas the actual content of the text field is read out as "accessibilityValue".

/cc @yjbanov
